src/DotNetLightning.Core/Channel/ChannelValidation.fs

```diff
@@ -146,19 +146,28 @@ module internal ChannelHelpers =
 module internal Validation =
 
     open DotNetLightning.Channel
-
+    let checkOurOpenChannelMsgAcceptable (conf: ChannelConfig) (msg: OpenChannel) =
+        Validation.ofResult(OpenChannelMsgValidation.checkFundingSatoshisLessThanMax msg)
+        *^> OpenChannelMsgValidation.checkChannelReserveSatohisLessThanFundingSatoshis msg
+        *^> OpenChannelMsgValidation.checkPushMSatLesserThanFundingValue msg
+        *^> OpenChannelMsgValidation.checkFundingSatoshisLessThanDustLimitSatoshis msg
+        *^> OpenChannelMsgValidation.checkMaxAcceptedHTLCs msg
+        *^> OpenChannelMsgValidation.checkFunderCanAffordFee (msg.FeeRatePerKw) msg
+        |> Result.mapError(InvalidOpenChannelError.Create msg >> InvalidOpenChannel)
```

Comment:
It may be good to add some message so that we can distinguish an error returned by `checkOurOpenChannelMsgAcceptable` and by `checkOpenChannelMsgAcceptable`
e.g. change this line to
```fs
       |> Result.mapError((@)["our open_channel msg is invalid"] >> InvalidAcceptChannelError.Create msg >> InvalidAcceptChannel)
```

Reply:
Done. I've added a similar message for their open_channel message too.